server/views.py

```diff
@@ -8,6 +8,7 @@
 
 from .notebooks.models import Notebook
 from .base.models import User
+from django.conf import settings
```

Comment:
This should go up with the other django imports. Typically these are grouped by: stdlib, 3rd-party, then local imports.

Reply:
Ah good point, we should probably figure out a way to check import order automatically. I think the cool kids might be using black for this...

Reply:
`isort` has a `--check-only` flag and can be used like `flake8` to check code. This could be added to CircleCI. Feel free to file an issue and I can add it in some time.

Reply:
Makes sense, I just checked and black (which I think we should also consider using) doesn't in fact do this. I'll file some an issue to try to get both implemented as part of our ci step.

Reply:
Filed #1552